west.yml

```diff
@@ -142,7 +142,7 @@ manifest:
       groups:
         - hal
     - name: hal_ambiq
-      revision: 705f1cbb1ccbdaac7613eb7b27c208c8f592e0c4
+      revision: c9bc1c5abeb8d567254ec2e682e8ec1253c71be3
       path: modules/hal/ambiq
```

Comment:
Generally you should use HAL PR# instead of the commit# of your development branch before the changes merging the mainline. I reviewed the HAL changes and merged it. Please update the revision to the latest commit of hal_ambiq main branch.